scenarios/python/console/call-center/call_center.py

```diff
@@ -18,14 +18,14 @@
 import user_config_helper
 
 # This should not change unless you switch to a new version of the Speech REST API.
-SPEECH_TRANSCRIPTION_PATH = "/speechtotext/v3.0/transcriptions"
+SPEECH_TRANSCRIPTION_PATH = "/speechtotext/v3.1/transcriptions"
 
 # These should not change unless you switch to a new version of the Cognitive Language REST API.
 SENTIMENT_ANALYSIS_PATH = "/language/:analyze-text";
 SENTIMENT_ANALYSIS_QUERY = "?api-version=2022-05-01";
 CONVERSATION_ANALYSIS_PATH = "/language/analyze-conversations/jobs";
-CONVERSATION_ANALYSIS_QUERY = "?api-version=2022-05-15-preview";
-CONVERSATION_SUMMARY_MODEL_VERSION = "2022-05-15-preview";
+CONVERSATION_ANALYSIS_QUERY = "?api-version=2022-10-01-preview";
+CONVERSATION_SUMMARY_MODEL_VERSION = "latest";
```

Comment:
what is the point of using latest, when CONVERSATION_ANALYSIS_QUERY and SPEECH_TRANSCRIPTION_PATH are static.

e.g. when v3.2 comes out, these existing clients will start using the new model version, but the old path and query, is that what you want to happen, or will that just make this more likely to break in the future?

Reply:
This change was provided by the Language team. It is likely that the Conversation Summary model version is updated frequently even when the API path remains the same, which would be hard to keep track of.